README.md

```diff
@@ -13,3 +13,14 @@ Check our [examples repo](https://github.com/aws-samples/aws-bootstrap-kit-examp
 As of today we expose only one global construct :
 
 * [AWS BootstrapKit](./source/aws-bootstrap-kit/README.md)
+
+## Clean up accounts
+
+AWS Bootstrap Kit creates accounts (e.g CICD, DNS, Staging) through AWS Organization. Behind the scene, we use CloudFormation Custom Resource and AWS Lambda to create to call [CreateAccount API](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CreateAccount.html). 
```

Comment:
..AWS Lambda to invoke the call ..

Reply:
Fixed the grammar. 

I shorten it to ...AWS Lambda to call...